pkg/controller/dataimportcron-controller_test.go

```diff
@@ -706,6 +706,64 @@ var _ = Describe("All DataImportCron Tests", func() {
 			Entry("has no tag", imageStreamName, 1),
 		)
 
+		It("Should succeed garbage collecting old version DVs", func() {
+			cron = newDataImportCron(cronName)
+			importsToKeep := int32(1)
+			cron.Spec.ImportsToKeep = &importsToKeep
+			cron.Annotations[AnnSourceDesiredDigest] = testDigest
```

Comment:
why do you need to set digest?

Reply:
You're right, we don't need it. Probably left over from the old test I wrote.
Fixed